blockmanager.go

```diff
@@ -845,7 +845,12 @@ func (b *blockManager) getCheckpointedCFHeaders(checkpoints []*chainhash.Hash,
 		currentInterval++
 	}
 
-	log.Infof("Attempting to query for %v cfheader batches", len(queryMsgs))
+	batchesCount := len(queryMsgs)
+	if batchesCount == 0 {
+		log.Info("No cfheader batches queries are needed")
```

Comment:
Don't think this log is necessary but not a blocker for me.

Reply:
removed